component/loki/relabel/relabel.go

```diff
@@ -229,3 +230,10 @@ func (c *Component) process(e loki.Entry) model.LabelSet {
 	}
 	return relabeled
 }
+
+func (c *Component) getRules() []*relabel.Config {
+	c.mut.RLock()
+	defer c.mut.RUnlock()
+
+	return c.rcs
+}
```

Comment:
Do we want to make this a capsule so:

1. It shows up as being less obscure when viewing the UI?
2. It prevents users from calling the function from River? (Which they can do for any exported function)

Reply:
Yes, agreed. Do you think that in the future we might have the need for an interface that can not only get all rules but do other stuff too? (eg. fetch first N rules, fetch only `keep` rules etc?)

Reply:
Marked it as a capsule type.

I'm thinking about whether we're okay to have the same type for all three namespaces.

Reply:
Oops, sorry, I typed that before I saw your reply :) Sharing the type probably makes sense.